packages/shared/src/components/RichTextEditor/plugins/LinkPlugin/FloatingLink/EditLinkModal/EditModal.tsx

```diff
@@ -0,0 +1,45 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconPen16, IconTrashBin16 } from '@frontify/fondue';
+import { useFloatingLinkUrlInput } from '@udecode/plate';
+import React from 'react';
+import { useGuidelineDesignTokens } from '../../../../../../hooks';
+import { FloatingLink } from '../FloatingLink';
+
+export const EditModal = () => {
```

Comment:
what will we do with the Modals / Plugins in fondue? Will they be removed?

Reply:
If not: Maybe we could reuse some parts of it such that we don't have two places to maintain?